hc/src/help.src

```diff
@@ -116,6 +116,7 @@
 {2.52 , 1, "Latoocarfian"}
 {2.53 , 1, "Mandelbrot Mix 4"}
 {2.54 , 1, "DivideBrot5"}
+{2.55 , 1, "BurningShip"}
```

Comment:
Use `"Burning Ship"` instead of `"WordsSmashedTogether"`

Reply:
I left this one as it created errors that I couldn't resolve.